tests/e2e/specs/auth-flow-proxy.test.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * WordPress dependencies
+ */
+import { deactivatePlugin, visitAdminPage } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { resetSiteKit } from '../utils';
+
+describe( 'Site Kit set up flow for the first time', () => {
+	afterEach( async () => {
+		await resetSiteKit();
```

Comment:
Not necessary since this is just a single test, this is covered by the bootstrap in `beforeAll` and `afterAll`.